ts/src/pro/binance.ts

```diff
@@ -1167,11 +1165,10 @@ export default class binance extends binanceRest {
         const url = this.urls['api']['ws'][type] + '/' + this.options[type]['listenKey'];
```

Comment:
not sure we're handling this 'ACCOUNT_UPDATE' update properly 
![image](https://github.com/ccxt/ccxt/assets/43336371/d51f10f7-81e8-43eb-ba48-de6ab1836d81)

Reply:
Because after it, I get
```
  USDT: { free: undefined, used: undefined, total: 23.5480726 },
```

Reply:
Yes, I ran into the same which thought was weird, but seems on purpose. The Balance object received from binance is the following and we only parse `wb` as the total balance and that's why `free` and `used` are undefined.
```
{
                          "a":"BTC",                // Asset
                          "wb":"122624.12345678",   // Wallet Balance
                          "cw":"100.12345678"       // Cross Wallet Balance
                        },
        ```

Reply:
@pcriadoperez I see, but I'm wondering if we could use this `bc` value 
![image](https://github.com/ccxt/ccxt/assets/43336371/4095ec0e-dd55-4ffd-8cc2-a76b0766d955)

Reply:
@carlosmiei I'm not sure we could... what would be formula?
In the past a user suggested to use `cw` but I believe it only worked for isolated positions.
Also I'd be a bit worried to use `bc` with that comment of `except PnL and commision`. We might not be returning the correct value.